lambdas/models/bulk_upload_report.py

```diff
@@ -28,6 +29,11 @@ class BulkUploadReport(BaseModel):
     uploader_ods_code: str = Field(alias=MetadataReport.UploaderOdsCode)
     reason: Optional[str] = Field(default="", alias=MetadataReport.Reason)
 
+    def get_registered_at_uploader_practice_status(self) -> str:
+        if self.pds_ods_code in PatientOdsInactiveStatus.list():
```

Comment:
We can convert this into a ternary statement:
`return self.pds_ods_code if self.pds_ods_code in PatientOdsInactiveStatus.list() else str(self.uploader_ods_code == self.pds_ods_code)`

It's quite a long line of code, I don't know how this'd be default-formatted by Python but for readability (at least from a Java perspective 😄 it'd be nice like this:

```
return self.pds_ods_code 
    if self.pds_ods_code in PatientOdsInactiveStatus.list() 
    else str(self.uploader_ods_code == self.pds_ods_code)
```